common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -712,11 +689,34 @@ def change_enrollment(strategy, user=None, is_dashboard=False, *args, **kwargs):
             except (
                 CourseDoesNotExistException,
                 ItemAlreadyInCartException,
-                AlreadyEnrolledInCourseException
+                AlreadyEnrolledInCourseException,
             ):
                 pass
             # It's more important to complete login than to
             # ensure that the course was added to the shopping cart.
             # Log errors, but don't stop the authentication pipeline.
-            except Exception as ex:
+            except Exception as ex:  # pylint: disable=broad-except
                 logger.exception(ex)
+
+
+@partial.partial
```

Comment:
This is unnecessary; we don't expect re-entry into the pipeline on this function.